### ISSUE TITLE
Bonus Database Table(s)

### ISSUE THREAD

**Issue by Fiskoal** [Thaewyn/operation-jungle-knight#13]
Suggestions for new database table/columns, to collect interesting data from player runs, such as:

- death data
  - current act/encounter at time of death
  - enemy the player was killed by
- victory data
  - run time
  - number of enemies slain

This thread should be used for suggestions for data that will not be important to the performance of the game, but rather just for the player's interest (and ours, of course). This would be used to just see general trends in game play to help with balancing, and making neat infographs.

**Comment by Thaewyn** [comment]
Yeah, definitely something we'll want to track. I think my design document had some rough data for a 'run history' of some kind, so we'll need to design out that schema.
